src/test/java/org/opensearch/security/test/helper/rest/RestHelper.java

```diff
@@ -174,12 +174,15 @@ public void cancelled() {
 
     public HttpResponse[] executeMultipleAsyncPutRequest(final int numOfRequests, final String request, String body) throws Exception {
```

Comment:
This function is not great and its used in only one place, can you move it up to `testParallelPutRequests` so others aren't tempted to us it?